js/snippetPreview.js

```diff
@@ -410,6 +419,10 @@ SnippetPreview.prototype.renderTemplate = function() {
 	} );
 
 	this.element = {
+		measure: {
```

Comment:
Disagree. These fields holds the element to calculate the measurements and are not the results of it.